ClosedXML_Tests/Excel/CalcEngine/TextTests.cs

```diff
@@ -73,6 +73,26 @@ public void Code_Value()
 
         [Test]
         public void Concat_Value()
+        {
+            Object actual = XLWorkbook.EvaluateExpr(@"Concat(""ABC"", ""123"")");
+            Assert.AreEqual("ABC123", actual);
+
+            actual = XLWorkbook.EvaluateExpr(@"Concat("""", ""123"")");
+            Assert.AreEqual("123", actual);
+
+            var ws = new XLWorkbook().AddWorksheet();
+
+            ws.FirstCell().SetValue(20)
+                .CellBelow().SetValue("AB")
+                .CellBelow().SetFormulaA1("=DATE(2019,1,1)")
+                .CellBelow().SetFormulaA1("=CONCAT(A1:A3)");
+
+            actual = ws.Cell("A4").Value;
+            Assert.AreEqual("20AB43466", actual);
```

Comment:
Could you please add a similar test case for Concatenate function with the expected result "20"? It works this way, but I think it would be nice to have it covered with a test.

Reply:
OK, but that exact same test for `CONCATENATE` would and should throw an exception.

Reply:
Huh. I misinterpreted Excel's behavior. Turned out, it treats such expressions like array functions:
![image](https://user-images.githubusercontent.com/19576939/55324940-97316300-5494-11e9-90a4-189d8b4806e1.png)
(all B-cells have the same formula here)

Reply:
Added test for this.